src/api/github/getClient.ts

```diff
@@ -46,22 +40,18 @@ export async function getClient(type: ClientType, org: string | null) {
 
     return _getUserClient();
   } else {
-    if (!process.env.GH_APP_SECRET_KEY) {
-      throw new Error('GH_APP_SECRET_KEY not defined');
-    }
-    if (!process.env.GH_APP_IDENTIFIER) {
-      throw new Error('GH_APP_IDENTIFIER not defined');
-    }
     if (org == null) {
       throw new Error(
         'Must pass org to `getClient` if getting an app scoped client.'
       );
     }
 
-    const auth: AppAuthStrategyOptions = {
-      appId: Number(process.env.GH_APP_IDENTIFIER),
-      privateKey: process.env.GH_APP_SECRET_KEY?.replace(/\\n/g, '\n'),
-    };
+    const auth = GH_APP_AUTH_OPTIONS.get('__tmp_org_placeholder__');
+    if (auth === undefined) {
+      throw new Error(
```

Comment:
Maybe worthwhile having a test that covers this error? Or at least leaving a TODO for the future?

Reply:
https://github.com/getsentry/eng-pipes/pull/509

Reply:
Chaining this PR on that one. 👍